### PR TITLE
adjust action notify enum

### PR DESCRIPTION
**Pull request by justinzhou93**

### CODE REVIEW

File: pallets/automation-time/src/lib.rs [hunk 1]
```diff
@@ -71,11 +71,17 @@ pub mod pallet {
...
+	pub struct NotificationAction {
+		message: Vec<u8>,
+	}
+
 	/// The enum that stores all action specific data.
 	#[derive(Debug, Eq, PartialEq, Encode, Decode, TypeInfo)]
 	#[scale_info(skip_type_params(T))]
 	pub enum Action {
-		Notify(Vec<u8>),
+		Notify(NotificationAction),
```
Comment (rhuttman) [inline pallets/automation-time/src/lib.rs:84]: hmm I was just thinking of doing this in-line, but i guess a struct is fine

**Review by rhuttman — COMMENTED** [review]
I would hope at least 1 test failed due to this

File: pallets/automation-time/src/lib.rs (hunk 1 above)
Comment (justinzhou93) [reply pallets/automation-time/src/lib.rs:84]: curious how i even do this inline. looked into it a bit and it looks like structs cannot be anonymous

Comment (rhuttman) [reply pallets/automation-time/src/lib.rs:84]: same thing as the events right below it

**Review by rhuttman — APPROVED** [review]

**Comment by justinzhou93** [comment]
code coverage is wrong, since it does it based on lines, not on what code has been run

**Pull request merged**